frontend/device/generic/device.lua

```diff
@@ -140,6 +138,7 @@ function Device:resume()
     end
     self.screen_saver_mode = false
     self.powerd:refreshCapacity()
+    self.powerd:wakeUp()
```

Comment:
Do you consider `self.powerd:wakeUp()` a hardware operation? If so, we should put it at the beginning of `Device:resume` since the generic resume method is pure UI (i.e. software) code.

Either way, we should document what is expected to be done in `wakeUp` and `sleep` functions :)